feat: expose the full config to handlers (#509)
mkdocstrings/mkdocstrings: docs/handlers/overview.md

thanks to namespace packages. For more information about namespace packages,
[see their documentation](https://packaging.python.org/guides/packaging-namespace-packages/).

> TIP: **TL;DR - Project template for handlers.**

Reviewer: ```suggestion
TIP: **TL;DR - Project template for handlers.**  
```

These arguments are all passed as keyword arguments, so you can ignore them
by adding `**kwargs` or similar to your signature. You can also accept
additional parameters: the handler's global-only options and/or the root
config options. This gives flexibility and access to the mkdocs config, mkdocstring

> config and etc.. You should never modify the root config but can be used to get
> information about the mkdocs instance such as where the current `site_dir` lives.

Reviewer: ```suggestion
config etc.. You should never modify the root config but can use it to get
information about the MkDocs instance such as where the current `site_dir` lives.
```
